DMTN-107.tex

```diff
@@ -67,14 +68,18 @@ \subsection{Alert Production}\label{ssec:req_ap}
 
 \subsection{Alert Packet Contents}\label{ssec:req_apc}
 
-The Data Products Definition Document \citeds{LSE-163} specifies that the alert packets include {\em at least} a 12 month history, and cutouts of the template and difference images (Section 3.5.1).
-The 12 month history is not a true requirement, and should be interpreted as applying only after 12 months of LSST operations.
+\citeds{LSE-61} contains a formal requirement that the alert packets include {\em at least} a 12 month history (if available), as well as cutouts of the template and difference images\dmreq{0274}.
+The 12 month history would not be available in LOY1, and the requirement adequately accounts for this, so it would not be violated in LOY1.
 Alerts issued without image cutouts would not violate any requirements, but would be a non-standard packet. 
```

Comment:
DMS-REQ-0274 also requires cutouts from the template image and the difference image.

Reply:
Yes... but this version does adequately cite 0274 for those cutouts?

Reply:
The statement "Alerts issued without image cutouts would not violate any requirements" is not, I think, correct.

Reply:
Oooh right... when I was just citing LSE-163, it wasn't a violation... but now that we cite LSE-61's REQ-0274, it is... *and* issuing alert packets without cutouts is being considered, but in that case the cutouts are still provided just via a different mechanism...
So yes, I think the most appropriate thing to say is "Alerts issued without corresponding image cutouts being made available, as part of the alert packet or via an alternative distribution mechanism, would violate this requirement."
I'll make that update...

Reply:
(Why are we discussing alerts without cutouts in the context of what happens in LOY1?)

Reply:
Because some of the options presented would not lead to cutouts (e.g., if no templates are used, or if catalog differencing is done).

Reply:
Re. above, only the "consistency" paragraph in Section 3.2 needed a mild update, all else was already consistent with these updates to Section 2.2.